Dockerfile.emg

```
@@ -0,0 +1,32 @@
+ARG DISTRO_NAME
+ARG DISTRO_VERSION
+FROM $DISTRO_NAME:$DISTRO_VERSION
+
+LABEL maintainer="javier@netmanagers.com.ar"
+
+ARG SALT_INSTALL_METHOD
+ARG SALT_VERSION
+ARG PYTHON_VERSION
+ARG EXTRA_PACKAGES=""
+
+ARG PKGS="udev git net-tools sudo curl openssh-server openssh-clients which findutils glibc-langpack-en $EXTRA_PACKAGES"
+
+RUN echo 'ACCEPT_KEYWORDS="~amd64"' | tee -a /etc/portage/make.conf
```

We should avoid this.

> > * Found the solution here, i.e.
> >   
> >   * `echo 'ACCEPT_KEYWORDS="~amd64"' | tee -a /etc/portage/make.conf`
> > ```
> 
> Yeah, common pitfall, and a logical one too, however, from now entire "world" (all system packages + your installed packages) will now use the unstable branch (amd64 == stable / ~amd64 == unstable) and force an awful lot of rebuilds, if the package manager even is able to resolve it.

Replacing it with something based on:

* https://wiki.gentoo.org/wiki/Knowledge_Base:Accepting_a_keyword_for_a_single_package#Manual_resolution

Perhaps:

```suggestion
RUN mkdir --p /etc/portage
RUN echo "=app-admin/salt ~amd64" >> /etc/portage/package.accept_keywords
```

* Probably don't need the `mkdir --p /etc/portage` but just taking that from the referenced wiki page

---

You are right, you should not need it, but it doesn't hurt either. However it could cover up a case where it is not available (which it should, because for example the system profile is defined/linked in there)
Rest looks fine.